pupa/filters.py

```diff
@@ -0,0 +1,12 @@
+import re
+
+
+_bill_id_re = re.compile(r'([A-Z]*)\s*0*([-\d]+)')
+_mi_bill_id_re = re.compile(r'(SJR|HJR)\s*([A-Z]+)')
+
+
+def fix_bill_id(bill_id):
```

Comment:
ideally this would move out of pupa altogether now, but it is no longer called by default